pkgs/applications/office/gtg/default.nix

```diff
@@ -0,0 +1,74 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, python3
+, python3Packages
```

Comment:
I would just bring in one or the other, but not both. Makes overriding easier

Reply:
makes sense, is changed